### ISSUE TITLE
可否在README里面把构建方法写的更详细一点？

### ISSUE THREAD

**Issue by rockswang** [skywind3000/PyStand#27]
本人C++/VC盲，大佬们默认的常识也不懂，可否麻烦把构建过程写的更详细一些，比如VC版本需求，详细命令行，可能出现的错误等等，多谢大佬们啦！

**Comment by skywind3000** [comment]
可以的，有空我补充下。

**Comment by myd7349** [comment]
我去年七月份的时候写过一篇文章，里边有一段介绍如何从源码编译 PyStand。我刚刚将其发布为 gist 了，后续会进一步完善一下：https://gist.github.com/myd7349/9f7c6334e67d1aee68a722a15df4a62a

**Comment by myd7349** [comment]
对 CMake 的基础用法又做了补充：https://gist.github.com/myd7349/9f7c6334e67d1aee68a722a15df4a62a#pystand-%E5%AE%9A%E5%88%B6

**Comment by skywind3000** [comment]
@myd7349 ，打包成唯一可执行可以尝试 Enigma Virtual Box 啊。

**Comment by myd7349** [comment]
> @myd7349 ，打包成唯一可执行可以尝试 Enigma Virtual Box 啊，将整个 PyStand 目录做成一个可执行。

好。我回头补充一下。
